docs/evm-sidechain/configure-the-bridge-client.md

```diff
@@ -0,0 +1,141 @@
+
+- Server
+- Logger
+- Database (TypeORM)
+- EVM
+- XRPL
+- XUMM
+
+These configuration files use the `buildConfig()` function, taking an object with the configuration properties and a second optional parameter to perform validations. Configuration properties can have a static value or an object specifying values for each environment. The following environments are available:
+
+- `default`
+- `production`
+- `development`
+- `test`
+- `staging`
+
+Configuration values can be constants, environment variables or AWS Secrets. Environment variables are defined in a .env file in the root of the backend. AWS Secrets add an extra layer of customisation and security. To load AWS secrets, the `AWS_REGION` and `AWS_SECRET_ID` environment variables must be defined.
+
+**Warning:** Never use constants for sensitive data, such as keys or secrets in production environments.
+
+
+### Server Options
+
+| Key | Description | EVN Key | AWS Secret | Example |
+|-----|-------------|---------|------------|---------|
+| port | Server port to receive requests. | `APP_PORT` | | `3000` |
+| secretKey | Secret key used to generate JWT tokens. | `APP_JWT_KEY` | | `"AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="` |
+| enableSwagger | Enable Swagger. | | | `true` |
+| enableCor | Enable CORS. | | | `true` |
+
+### Logger Options
+
+| Key | Description | EVN Key | AWS Secret | Example |
+|-----|-------------|---------|------------|---------|
+| logLevel | Log level used by the server logger. | | | `"info"` |
+|logFileName | File name to store logs. | | | `"app.log"` |
+
+### Database (TypeORM) Options
+
+| Key | Description | EVN Key | AWS Secret | Example |
+|-----|-------------|---------|------------|---------|
+| host | Database host. | `DB_HOST` | `DB_HOST` | `"db"` |
+| port | Database port. | `DB_PORT` | `DB_PORT` | `5432` |
+| username | Database username. | `DB_USER` | `DB_USER` | `"db_user"` |
+| password | Database password. | `DB_PASSWORD` | `DB_PASSWORD` | `"db_password"` |
+| database | Database name. | `DB_DATABASE` | `DB_DATABASE` | `"db_database"` |
+
+**Note:** This configuration also extends [Nest TypeORM properties](https://docs.nestjs.com/techniques/database).
+
+### EVM Option
+
+| Key | Description | EVN Key | AWS Secret | Example |
+|-----|-------------|---------|------------|---------|
+| server | EVM chain RPC. | | | `"https://rpc-evm-sidechain.xrpl.org"` |
+
+### XRPL Option
+
+| Key | Description | EVN Key | AWS Secret | Example |
+|-----|-------------|---------|------------|---------|
+| server | XRPL chain WebSocket. | | | `"wss://sidechain-net1.devnet.rippletest.net"` |
+
+### XUMM Options
+
+| Key | Description | EVN Key | AWS Secret | Example |
+|-----|-------------|---------|------------|---------|
+| appKey | XUMM app key. | `XUMM_API_KEY` | `XUMM_API_KEY` | `"aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa"` |
+| appSecret | XUMM app secret. | `XUMM_SECRET_KEY` | `XUMM_SECRET_KEY` | `"aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa"` |
```

Comment:
How does one get a XUMM app key & app secret? A link to the relevant documentation would be nice.